cli/pg.go

```diff
@@ -160,11 +166,22 @@ func runPgRestore(args *docopt.Args, client controller.Client, config *runConfig
 		defer bar.Finish()
 		config.Stdin = bar.NewProxyReader(config.Stdin)
 	}
-	return pgRestore(client, config)
+
+	return pgRestore(client, config, jobs)
 }
 
-func pgRestore(client controller.Client, config *runConfig) error {
-	config.Args = []string{"pg_restore", "-d", config.Env["PGDATABASE"], "--clean", "--if-exists", "--no-owner", "--no-acl"}
+func pgRestore(client controller.Client, config *runConfig, jobs int) error {
+	// Provision a volume at /data to stream the dump to.
+	if jobs > 1 {
+		config.Data = true
+		config.Args = []string{
+			"bash",
+			"-c",
+			fmt.Sprintf("set -o pipefail; cat > /data/temp.dump && pg_restore -d %s -n public --clean --if-exists --no-owner --no-acl --jobs %d /data/temp.dump", config.Env["PGDATABASE"], jobs),
```

Comment:
I don't believe `pg_restore` creates temporary files (though I do believe `pg_dump` does if you use certain formats and stream to stdout).

We are writing the file to `/data` because it may be too big for the container root filesystem and `/tmp` may be on `tmpfs` and the dump could be bigger than available RAM.